source/appModules/azardi-2_0.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: UTF-8 -*-
+#appModules/azardi-2_0.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 Noelia Ruiz Martínez
```

Comment:
Could you please update the copyright to match https://github.com/nvaccess/nvda/wiki/Copyright-headers